### ISSUE TITLE
How to create a hotspot without password? (No password required)?

### ISSUE THREAD

**Issue by nadavb** [prahladyeri/hotspotd#50]
When I do: `sudo hotspotd configure`, it asks for password.
I would like to create a hotspot with SSID, but with no need for a password.

**Comment by prahladyeri** [comment]
Yep, you can edit the hotspot configuration through `run.conf`. I'd still recommend you to setup a password though, a fully transparent wifi hotspot is insecure as anyone with a wifi device can use your hotspot without authorization.
